BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/TransactionDocs/Models/GiniTransaction.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  GiniTransaction.swift
+//
+//  Copyright © 2025 Gini GmbH. All rights reserved.
+//
+
+/**
+ A representation of a transaction done via PhotoPayment containing attached documents if they exist.
+ */
+public struct GiniTransaction {
+
+    /**
+     The unique identifier for the transaction.
```

Comment:
[Suggestion]: I suggest to keep the comment short like `  A unique identifier for the transaction.` and the second line can be removed